app/workers/education_form/templates/header.erb

```diff
@@ -1,9 +1,10 @@
 *INIT*
-<%= @applicant.veteranFullName.first.upcase %>
-<%= @applicant.veteranFullName.middle&.upcase %>
-<%= @applicant.veteranFullName.last.upcase %>
-<%= @applicant.veteranSocialSecurityNumber&.gsub(/[^\d]/,'') %>
-<%= @applicant.veteranSocialSecurityNumber&.gsub(/[^\d]/,'') %>
-V<%= @record.form_type %>
+<% %w(first middle last).each do |name| -%>
+<%= applicant_name&.public_send(name)&.upcase %>
+<% end -%>
+<% [@applicant.veteranSocialSecurityNumber, applicant_ssn].each do |ssn| -%>
+<%= ssn&.gsub(/[^\d]/, '') %>
+<% end -%>
```

Comment:
I'd prefer to keep this explicit, rather than making it _look_ dynamic. This is a very fixed header.